opencti-platform/opencti-front/src/private/components/analyses/reports/ReportDetails.jsx

```diff
@@ -122,6 +122,41 @@ const ReportDetailsComponent = (props) => {
   ).filter(
     (relatedContainerEdge) => relatedContainerEdge.node.id !== report.id,
   );
+
+  const entitiesDidstributionDataSelection = [
+    {
+      label: '',
+      attribute: 'entity_type',
+      date_attribute: 'first_seen',
+      perspective: 'relationships',
+      isTo: true,
+      filters: {
+        mode: 'and',
+        filters: [
+          {
+            key: 'relationship_type',
+            values: [
+              'object',
+            ],
+            operator: 'eq',
+            mode: 'or',
+          },
+        ],
+        filterGroups: [],
+      },
+      dynamicFrom: {
+        mode: 'and',
+        filters: [],
+        filterGroups: [],
+      },
+      dynamicTo: {
+        mode: 'and',
+        filters: [],
+        filterGroups: [],
+      },
```

Comment:
you can `import { emptyFilterGroup } from '../../../utils/filters/filtersUtils';` to simplify this part

```suggestion
      dynamicFrom: emptyFilterGroup,
      dynamicTo: emptyFilterGroup,
```